app/views/rooms/cant_create_rooms.html.erb

```diff
@@ -17,10 +17,10 @@
 
 <div id="cant-create-room-wrapper" class="background h-100 cant-create-room-wrapper">
     <div class="container h-100">
-        <div class="row h-100 align-items-center">
+        <div class="row h-100 pt-7 align-items-center">
             <div class="col-12">
                 <div class="row">
-                    <div class="col-5">
+                    <div class="col-lg-5 col-sm-12">
```

Comment:
We may actually want to hide the profile portion when on mobile, which would make it easier for a user on mobile to join a room